doc/source/conf.py

```diff
@@ -20,6 +20,8 @@
 from numpydoc.docscrape import NumpyDocString
 from sphinx.ext.autosummary import _import_by_name
 
+import pandas_sphinx_theme  # noqa
```

Comment:
why is this needed? may be add a comment in the code to explain?

Reply:
This might actually not be needed, will try out (I was assuming it is needed, because in the init of the package we have some sphinx extensions added, but probably sphinx imports the package anyway if you set it as the html theme)

Reply:
Doesn't seem needed. Removed.